exporter/prometheusexporter/README.md

```diff
@@ -35,3 +35,71 @@ exporters:
     resource_to_telemetry_conversion:
       enabled: true
 ```
+
+## Automatic Metrics Renaming
+
+> **Warning**
+>
+> This feature must be enabled with [feature gate](https://github.com/open-telemetry/opentelemetry-collector/tree/main/service/featuregate) `exporter.prometheus.AutomaticRename`. It is disabled by default (alpha stage).
+>
+> ```shell-session
+> $ otelcol --config=config.yaml --feature-gates=exporter.prometheus.AutomaticRename
+> ```
+
+[OpenTelemetry's metric semantic convention](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/README.md) is not compatible with [Prometheus' own metrics naming convention](https://prometheus.io/docs/practices/naming/). The Prometheus Exporter automatically renames the metrics to ensure they follow Prometheus' naming convention.
+
+List of transformations performed on OpenTelemetry metrics:
+
+| Case                                                     | Transformation                                                                                                                   | Example                                                                                     |
+| -------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------- |
+| Standard unit                                            | Convert the unit from [Unified Code for Units of Measure](http://unitsofmeasure.org/ucum.html) to Prometheus standard and append | `system.filesystem.usage` with unit `By` ==> `system_filesystem_usage_bytes`                |
+| Non-standard unit (unit is surrounded with `{}`)         | Drop the unit                                                                                                                    | `system.network.dropped` with unit `{packets}` ==> `system_network_dropped`                 |
+| Non-standard unit (unit is **not** surrounded with `{}`) | Append the unit, if not already present, after sanitization (all non-alphanumeric chars are dropped)                             | `system.network.dropped` with unit `packets` ==> `system_network_dropped_packets`           |
+| Percentages (unit is `1`)                                | Append `_ratio` (for gauges only)                                                                                                | `system.memory.utilization` with unit `1` ==> `system_memory_utilization_ratio`             |
```

Comment:
Many Otel receivers improperly use unit `1` for counting objects, while it's reserved for fractions and ratios.

To identify these misuses of the `1` as a unit and to avoid appending `_ratio` where it's not expected, we check that the metric is a gauge, because ratios and fractions are always gauges.

In the meantime, I also created many issues to fix these improper units in receivers.

Reply:
Could ratios be counters (e.g. fraction_of_work_done_ratio)?  I'm OK with it as a temporary restriction for practicality, but I'd prefer to not keep it around longer-term, as we shouldn't prevent things allowed by the OpenMetrics spec.  If there is an umbrella tracking issue for correcting the use of `1`, it would be helpful to link that by the "Append _ratio for ratio Gauges" comment.

Reply:
There is no such thing as metrics suffixed with `_ratio_total` in Prometheus. It's either `_ratio` or `_total`, it really can't be both. Same for OpenMetrics. I think we're safe here ;-)

Reply:
You're actually right: theoretically, it's possible.

However, `_total` indicates a monotonic counter, which keeps increasing. `_ratio` indicates a value comprised between 0 and 1. This means the metric is a function with a limit of 1 as it approaches infinite. It also means that its derivative has a limit of zero when it approaches infinite. This means that the `rate()` of this metric will be always zero after a certain time. Which is absurd.

Not appending `_ratio` to such metrics will not be a problem to any "real life" scenario, and will allow us to cover many cases where the unit was improperly specified as `1`.

I'm adding a note to the code as you suggested.